packages/babel-plugin-transform-optional-chaining/src/index.js

```diff
@@ -123,7 +125,7 @@ export default function({ types: t }) {
           return;
         }
 
-        optional(path, findReplacementPath(path), this.opts.loose);
+        optional(path, findReplacementPath(path), loose);
```

Comment:
`options` is in scope of the `optional` function, so this third argument isn't needed anymore.